packages/compass-collection/src/components/collection-tab-provider.tsx

```diff
@@ -5,13 +5,15 @@ import type { CollectionSubtab } from '@mongodb-js/compass-workspaces';
 
 export interface CollectionTabPlugin {
   name: CollectionSubtab;
-  component: HadronPluginComponent<CollectionTabPluginMetadata, any, any>;
+  Provider: HadronPluginComponent<CollectionTabPluginMetadata, any, any>;
+  Content: React.FunctionComponent<CollectionTabPluginMetadata>;
+  Header: React.FunctionComponent;
```

Comment:
Just a small naming nit: if we're going with capitalized naming scheme here, we might want to rename the other types to match (like queryBar to QueryBar below), otherwise maybe let's stick with lowercase. No preferences on what to choose if it's consistent

Reply:
fixed in [b750ce1](https://github.com/mongodb-js/compass/pull/6267/commits/b750ce1e5443493836d4fa103b2333efc1ccfbf2)